### PR TITLE
Replace some more Uint32 values with formal typedefs

### LINKED ISSUE #9812: Some bitflags still need a typedef.
Right now SDL_Surface::flags is a Uint32. It should get the same typedef update (and wikiheaders magic) that other bitflags did.

### PR DESCRIPTION
**Pull request by icculus**
This replaces some Uint32 values I've stumbled upon into formal bitflag typedefs, like we've done for other things.

Fixes #9812.

### CODE REVIEW

File: include/SDL3/SDL_mouse.h [hunk 1]
```diff
@@ -83,6 +83,37 @@ typedef enum SDL_MouseWheelDirection
     SDL_MOUSEWHEEL_FLIPPED    /**< The scroll direction is flipped / natural */
 } SDL_MouseWheelDirection;
 
+/**
+ * A bitmask used when testing if a mouse's buttons are pressed.
```
Comment (slouken) [inline include/SDL3/SDL_mouse.h:87]: This comment needs to be updated to refer to the type, like the other flag typedefs.

File: include/SDL3/SDL_pen.h [hunk 2]
```diff
@@ -107,7 +107,13 @@ typedef enum SDL_PenAxis
 #define SDL_PEN_TIP_INK    SDL_PEN_FLAG_INK_BIT_INDEX     /**< Regular pen tip (for drawing) touched the surface */
 #define SDL_PEN_TIP_ERASER SDL_PEN_FLAG_ERASER_BIT_INDEX  /**< Eraser pen tip touched the surface */
 
-/* Pen capabilities reported by SDL_GetPenCapabilities */
+/**
+ * Pen capabilities reported by SDL_GetPenCapabilities.
+ *
+ * \since This datatype is available since SDL 3.0.0.
+ */
+typedef Uint64 SDL_PenCapabilityFlags;
```
Comment (slouken) [inline include/SDL3/SDL_pen.h:115]: Uint32? Or did you mean to change it?

File: include/SDL3/SDL_test_common.h [hunk 3]
```diff
@@ -45,6 +45,7 @@
 #define DEFAULT_WINDOW_HEIGHT 480
 #endif
 
+typedef Uint32 SDLTest_VerboseFlags;
```
Comment (slouken) [inline include/SDL3/SDL_test_common.h:48]: I don't think we need to bother with this for the testing code.

File: src/events/SDL_mouse_c.h [hunk 4]
```diff
@@ -154,9 +154,6 @@ extern void SDL_SetMouseFocus(SDL_Window *window);
 /* Update the mouse capture window */
 extern int SDL_UpdateMouseCapture(SDL_bool force_release);
 
-/* Get the current mouse button state for a mouse */
-Uint32 SDL_GetMouseButtonState(SDL_Mouse *mouse, SDL_MouseID mouseID, SDL_bool include_touch);
```
Comment (slouken) [inline src/events/SDL_mouse_c.h:158]: Is this not called outside of SDL_mouse.c?

File: include/SDL3/SDL_pen.h (hunk 2 above)
Comment (icculus) [reply include/SDL3/SDL_pen.h:115]: Whoops, left that in to check for compiler warnings, good catch!

File: src/events/SDL_mouse_c.h (hunk 4 above)
Comment (icculus) [reply src/events/SDL_mouse_c.h:158]: It's a duplication definition (it's in the public header, too).

File: include/SDL3/SDL_mouse.h (hunk 1 above)
Comment (icculus) [reply include/SDL3/SDL_mouse.h:87]: "A bitmask of pressed mouse buttons, as reported by SDL_GetMouseState, etc." ?

File: include/SDL3/SDL_pen.h (hunk 2 above)
Comment (icculus) [reply include/SDL3/SDL_pen.h]: Fixed in 1936e91d85dcf4087ac82769aa27cf3f7630dcb4.

File: include/SDL3/SDL_test_common.h (hunk 3 above)
Comment (icculus) [reply include/SDL3/SDL_test_common.h:48]: Probably, but it doesn't hurt.

File: include/SDL3/SDL_mouse.h (hunk 1 above)
Comment (slouken) [reply include/SDL3/SDL_mouse.h:87]: Yeah, that sounds good.

Comment (icculus) [reply include/SDL3/SDL_mouse.h]: Fixed in de32eb906fde7549822fc5a928f1280807945f97.

**Review by slouken — APPROVED** [review]

**Pull request merged**